Whack-a-mole/main.js

```diff
@@ -0,0 +1,47 @@
+const holes = document.querySelectorAll(".hole");
+const scoreBoard = document.querySelector(".score");
+const moles = document.querySelectorAll(".mole");
+let lastHole;
+let timeUp = false;
+let score = 0;
+
```

Comment:
According to https://github.com/rolling-scopes-school/tasks/blob/master/tasks/stage-0/projects.md#task-10-whack-a-mole-40, please add local storage saving and progressively more difficult levels

Reply:
done